install/actualbudget-install.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: MickLesk (CanbiZ)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://actualbudget.org/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Actual Budget"
+cd /opt
+RELEASE=$(curl -fsSL https://api.github.com/repos/actualbudget/actual/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+NODE_VERSION="22"
+install_node_and_modules
+mkdir -p /opt/actualbudget-data/{server-files,upload,migrate,user-files,migrations,config}
+chown -R root:root /opt/actualbudget-data
+chmod -R 755 /opt/actualbudget-data
+
+cat <<EOF >/opt/actualbudget-data/config.json
+{
+  "port": 5006,
+  "hostname": "::",
+  "serverFiles": "/opt/actualbudget-data/server-files",
+  "userFiles": "/opt/actualbudget-data/user-files",
+  "trustedProxies": [
+    "10.0.0.0/8",
+    "172.16.0.0/12",
+    "192.168.0.0/16",
+    "127.0.0.1/32",
```

Comment:
```suggestion
    "127.0.0.0/8",
```
Would be technically correct, as the whole 127 subnet is reserved for loopback